### PR TITLE
add portis

### PR DESCRIPTION
**Pull request by andrewdamelio**
relates to #2035 

- Add Portis wallet
- integrations with SDK
- misc type fixes

**NOTE:**
- Currently, when using Portis with a local node (i.e 104) you will see the below errors, expected, but looking into a way to avoid them.

- If your interested, you can access your Portis wallet at https://wallet.portis.io/. **FYI -  You won't see balances related to activity on custom nodes.**

### CODE REVIEW

File: packages/augur-ui/package.json [hunk 1]
```diff
@@ -99,6 +100,7 @@
     "shifty": "2.6.1",
     "speedomatic": "2.1.5",
     "trezor-connect": "7.0.3",
+    "web3": "^1.0.0-beta.55",
```
Comment (bthaile) [inline packages/augur-ui/package.json]: don't use automatic upgrades remove `^`

Comment (andrewdamelio) [reply packages/augur-ui/package.json]: 👍 updated

File: packages/augur-ui/src/modules/auth/actions/login-with-portis.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import { ACCOUNT_TYPES } from "modules/common/constants";
+import { AppState } from "store";
+
+// TODO find home for all wallet API keys
+const PORTIS_API_KEY = "b67817cf-8dd0-4116-a0cf-657820ddc019";
+
+const getPortisNetwork = (networkId): false | string | INetwork  => {
+  const myPrivateEthereumNode = {
+    nodeUrl: "http://localhost:8545",
+    chainId: "104",
```
Comment (bthaile) [inline packages/augur-ui/src/modules/auth/actions/login-with-portis.ts:20]: we actually have many private node chainIds. See `augur/packages/augur-artifacts/src/addresses.json`
don't use anything with augur-node. use `getNetworkId()` in contractCalls.ts

**Review by bthaile — CHANGES_REQUESTED** [review]
a few comments for changes.

File: packages/augur-ui/src/modules/auth/actions/login-with-portis.ts (hunk 2 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/auth/actions/login-with-portis.ts:20]: ahh that makes more sense!

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**